src/features/classification/ClassifiedBadge.scss

```diff
@@ -1,17 +1,5 @@
-@import '../../styles/variables';
-@import './mixins';
-
-.bdl-ClassifiedBadge,
-.bdl-ClassifiedBadge:hover {
-    @include bdl-ClassificationBadge;
-
-    background-color: $bdl-yellorange-10;
-    border-color: $bdl-yellorange;
-}
-
-.bdl-ClassifiedBadge-name {
-    @include bdl-ClassificationBadge-name;
-
-    color: $bdl-gray !important; // To inhibit overriding
-    text-transform: uppercase;
+.bdl-ClassifiedBadge {
+    .bdl-SecurityBadge-name {
+        margin-left: 5px;
```

Comment:
Is this margin not needed for space between icon and name?

Reply:
Couldn't find a good solution for this. The icons seems to have inconsistent whitespace around them (`IconAlertDefault` has a lot of it), so you basically have to tweak the margin manually for each. An alternative  would be to set a negative margin on `IconAlertDefault` so that all badges have that 5px spacing by default.